### DISCUSSION TITLE
No "Do not perform lifting sequence for layers with same Z positioning" when generating calibration file for Anycubic M3 Premium

### DISCUSSION THREAD

**Discussion by fcollingwood** [sn4k3/UVtools#706] [Q&A] [answered]
### System

```Text
UVtools v3.13.3 X64
Operative system: Microsoft Windows 10.0.22000 X64
Processor: Intel(R) Xeon(R) W-3223 CPU @ 3.50GHz
Processor cores: 16
Memory RAM: 148.56 / 255.65 GB
Runtime: win10-x64
Framework: .NET 6.0.16
AvaloniaUI: 0.10.19
OpenCV: 4.7.0

Sreens, resolution, working area, usable area:
1: 3440 x 1440 @ 100% (Primary) (On this)
    WA: 3440 x 1392    UA: 3440 x 1392
2: 3440 x 1440 @ 100%
    WA: 3440 x 1392    UA: 3440 x 1392


Path:       C:\Program Files\UVtools\
Executable: C:\Program Files\UVtools\UVtools.exe
Loaded file: UV Tools RERF.pm3r [Version: 517] [Class: PhotonWorkshopFile]
```

### Printer and Slicer

- Printer: Anycubic Photon M3 Premium
- Slicer: N/A

### Description of the bug

Expecting to be able to select "Do not perform lifting sequence for layers with same Z positioning" in the Multiple Exposures section, but it's not there - not even there but greyed out, just completely missing

### How to reproduce

Try to create a calibration file for an Anycubic M3 Premium

**Comment by sn4k3** [comment]
It's there for me, try to scroll the section down.

**Comment by fcollingwood** [comment]
Nope. Not there

![image](https://github.com/sn4k3/UVtools/assets/37040349/ad62192a-375e-4598-8819-b7edd71328e5)

**Comment by sn4k3** [comment, marked as answer]
Ah yes, sorry I was looking for other option.
That option is gone long time ago. To get same result set 0mm on 'Lift Height' option below. But that won't work for most formats, and that's why default is 0.10mm. So far only gcode printers can benefit from that. All other formats doesn't like the zero value

**Comment by fcollingwood** [comment]
That's a great pity, because non-gcode printers had no issues with the "Do not perform lifting sequence for layers with same Z positioning" option - I generated multiple exposure tests for my Phrozen Mighty 4Ks just a few weeks ago on an older version. Looks like I need to downgrade my installation back to what I was using

**Comment by sn4k3** [comment]
Phrozen uses a different firmware that it might work. But I guess it still does some minimal lift?

> Looks like I need to downgrade my installation back to what I was using

No... Read my reply again...

 `To get same result set 0mm on 'Lift Height' option below.`

In other words, "Do not perform lifting sequence for layers with same Z positioning" = Lift Height of 0mm
